### ISSUE TITLE
Stopped working. Exactly in the moment that goodbyedpi stopped with -9 option.

### ISSUE THREAD

**Issue by Wolchenok57** [hufrea/byedpi#131]
Worked for a while, then stopped. Here is my starting line:

/opt/byedpi/ciadpi-x86_64 -i 0.0.0.0 -p 8888 -c 30000 \
--hosts /opt/byedpi/yt.txt --timeout 3 --fake -1 --ttl 20 --tlsrec 3+s --split 3+s --disorder 7+s --md5sig --oob 3+s \
--hosts /opt/byedpi/gyt.txt --fake -1+s --ttl 1 \
--hosts /opt/byedpi/nyt.txt --timeout 3 --fake 1 --ttl 20 --tlsrec 3 --split 3 \
#yt,txt, gyt.txt and nyt.txt - files with domains on each line.

GoodbyeDPI with -7 option (idk what it does, i just installed it while ago while searching a way to do it on router level and now just changed "-9" to "-7") worked with youtube in my network on the main machine. Everyone on internet posting info how to fix GoodbyeDPI, but not byedpi.

And while I'm here, my smartphone youtube is also slow, and idk how to unslow it. I tried to play with youtubei.googleapis.com (it is the only domain in gyt.txt) but it is not as good as on PCs.

Или по-русски: работало-работало, да перестало. Строка запуска выше. Хелп. Плиз. Нужно, чтобы работало на всех пк в локальной сети, в том числе на линуксах. Настроено было на уровне роутера и работало приемлемо.
(И пока я тут: на смартфоне ютуб не работает нормально, пробовал тыкать домен youtubei.googleapis.com, он находится в gyt,txt, но не очень результат - приходится переподключать вайфай на мобильную сеть, но и мою мобильную сеть тоже начали замедлять (никак иначе не объяснить, что появились симптомы "не грузится ютуб"), так что лучше работает именно на вайфае, но иногда требует переподключения)
P.S. Мой провайдер - Мегафон. Город - Орёл (это в ~324 км от Москвы, туда же заворачивается зачем-то весь интернет-траффик, даже для сайтов, находящихся непосредственно в Орле).

**Comment by p-fpv** [comment]
У меня на ростелеком сейчас работает с `--disorder 2`, больше ничего не добавлял.  
`./ciadpi-x86_64 -i 127.0.0.1 -p 1080 --hosts ./list.txt --disorder 2`

**Comment by borodinalive** [comment]
Да, че т не работает. До этого работало с параметрами из примеров: --fake -1 --ttl 10 --auto=alert,sid_inv --fake -1 --ttl 5

**Comment by Wolchenok57** [comment]
В итоге я изменил свою строку запуска на следующее:
/opt/byedpi/ciadpi-x86_64 -i 0.0.0.0 -p 8888 -c 30000 \
--hosts /opt/byedpi/yt.txt  -F --split 2 --disorder -1+s \
--hosts /opt/byedpi/gyt.txt -F --split 2 --disorder -1+s --timeout 3 --fake 1 --ttl 20 --tlsrec 3 --split 3 \
--hosts /opt/byedpi/nyt.txt --timeout 3 --fake 1 --ttl 20 --tlsrec 3 --split 3
и оно заработало, но именно просто disorder не давал ничего. Пришлось еще раз поэкспериментировать, но наводка на особенность disorder по итогу помогла. Спасибо, в общем.
А по второму вопросу: что-то можно с мобильной версией сделать? Я еще раз с ней потыкался, смог заставить работать с перерывами, а не как было раньше "не буду грузиться, жди вечно" (я реально ждал более 20 минут, что произошло по случайности, и оно не грузило). Сейчас если подождать прогрузки менее 10 секунд, оно прогружается, но это все равно не дело, в сравнении с безупречной работой десктопа. (строка с gyt.txt отвечает за youtubei.googleapis.com, который, насколько я нагуглил, относится только к мобильным приложениям ютуба)

**Comment by v-fox** [comment]
> У меня на ростелеком сейчас работает с `--disorder 2`, больше ничего не добавлял.

"ТыТруба" действительно завелась с этим с полпинка (даже с http3/quic), как и все другие основные сайты. Но никак не могу заставить [nyaa.si](https://nyaa.si/?q=slime%20tensei) открываться с любыми настройками, что аж есть подозрение, что он сбрасывает на своей стороне с IP-диапазонов в РФ.
Ещё держу такие статичные правила `iptables`, которые работали до DPI:
```iptables
Chain REMOVE_CENSORSHIP (1 references)
target     prot opt source               destination         
DROP       6    --  0.0.0.0/0            0.0.0.0/0            tcp spt:80 STRING match  "Location: http://warning.rt.ru" ALGO name bm
DROP       6    --  0.0.0.0/0            0.0.0.0/0            tcp spt:80 flags:0x04/0x04 u32 "0x4=0x10000"
DROP       6    --  0.0.0.0/0            0.0.0.0/0            tcp spt:443 flags:0x04/0x04 u32 "0x4=0x10000"

```
Для игнорирования перенаправления и фальшивого сброса соединений.
+[dnscrypt-proxy](https://github.com/DNSCrypt/dnscrypt-proxy) либо DoH для обхода отравления DNS.

**Comment by mapron** [comment]
Same issue, stopped working (along with GoodbyeDPI and bol-van/zapret) 3 days ago; some video still were working till today. 
ISP: CORBINA-BROADBAND (Beeline).

Аналогичная проблема. Пробовал разные опции от GoodbyeDPI , zapret и ByeDPI. Результат один, что в Firefox, что в Chrome на попытке получения данных с *.googlevideo.com/videoplayback - идет обрыв связи. Различные сервисы DoH на результат не повлияли. Опции из этого треда тоже попробовал.
